### ISSUE TITLE
Date sort

### ISSUE THREAD

**Issue by offaxis** [arunghosh/react-time-line#69]
Maybe you can sort items before creating your array by day in your getFormattedData(items), like:
`items.sort((itemA, itemB) => {`
`if(moment(itemA.ts).isBefore(itemB.ts)) {`
`  return -1;`
`} else if(moment(itemB.ts).isBefore(itemA.ts)) {`
`  return 1;`
`} else {`
`  return 0;`
`}`
`}).forEach(....`

**Comment by arunghosh** [comment]
Yes this can be done. But I thought it is better the consumer decide on the sort order.
